<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObservationsDialog</class>
 <widget class="QWidget" name="ObservationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="LocationBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="spacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Observations</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contents" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="observationsLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="observationsListView">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="observationsButtons">
         <item>
          <widget class="QPushButton" name="addObservationButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>12</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteObservationButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="data">
        <attribute name="title">
         <string>Data</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Observer:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="observerComboBox"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_23">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Target:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="targetComboBox"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_16">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Optic:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="opticComboBox"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_25">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Limiting Mag:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="limitingMagComboBox">
               <item>
                <property name="text">
                 <string>2.0</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3.0</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4.0</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5.0</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6.0</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7.0</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8.0</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_17">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Ocular:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="ocularComboBox"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_26">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Seeing:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QComboBox" name="seeingComboBox">
               <item>
                <property name="text">
                 <string>Bad</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Poor</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Average</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Good</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Excellent</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_18">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Barlow:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="barlowComboBox"/>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_27">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Imager:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QComboBox" name="imagerComboBox"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_19">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Filter:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="filterComboBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Begin:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDateTimeEdit" name="beginDateTimeEdit">
               <property name="displayFormat">
                <string>yyyy/MM/dd hh:mm</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>End:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QDateTimeEdit" name="endDateTimeEdit">
               <property name="displayFormat">
                <string>yyyy/MM/dd hh:mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Accessories:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LogBookTextEdit" name="accessoriesTextEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>51</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>140</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="notes">
        <attribute name="title">
         <string>Notes</string>
        </attribute>
        <widget class="LogBookTextEdit" name="notesTextEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>501</width>
           <height>121</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>121</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogBookTextEdit</class>
   <extends>QTextEdit</extends>
   <header>LogBookTextEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
